### PR TITLE
Fix All the Docstrings

### PR DESCRIPTION
**Pull request by alecmerdler**
### Description

The `black` tool messed up all our docstrings ([known issue](https://github.com/psf/black/issues/473)). This PR runs [`docformatter`](https://github.com/myint/docformatter) against the entire Quay repo in order to fix it.

### Testing

The output of this command should be empty:
```shell
$ docformatter --pre-summary-newline --make-summary-multi-line --wrap-summaries 100 --wrap-descriptions 100 -r .
```

Fixes https://issues.redhat.com/browse/PROJQUAY-184
Related https://github.com/psf/black/issues/144

### CODE REVIEW

File: auth/context_entity.py [hunk 1]
```diff
@@ -12,10 +12,12 @@
 
 
 class ContextEntityKind(Enum):
-    """ Defines the various kinds of entities in an auth context. Note that the string values of
```
Comment (alecmerdler) [inline auth/context_entity.py]: This is a good example of what was fixed.

**Comment by kurtismullins** [comment]
@alecmerdler Is it possible to format these with line-width 100?

**Comment by alecmerdler** [comment]
> @alecmerdler Is it possible to format these with line-width 100?

Done

File: auth/oauth.py [hunk 2]
```diff
@@ -10,9 +10,8 @@
 
 
 def validate_bearer_auth(auth_header):
-    """ Validates an OAuth token found inside a basic auth `Bearer` token, returning whether it
-      points to a valid OAuth token.
-  """
+    """Validates an OAuth token found inside a basic auth `Bearer` token, returning whether it
```
Comment (kurtismullins) [inline auth/oauth.py]: This comment appears to have been missed

File: auth/permissions.py [hunk 3]
```diff
@@ -112,9 +112,8 @@ def _user_role_for_scopes(self, role):
         return self._translate_role_for_scopes(USER_ROLES, SCOPE_MAX_USER_ROLES, role)
 
     def _populate_user_provides(self, user_object):
-        """ Populates the provides that naturally apply to a user, such as being the admin of
-        their own namespace.
-    """
+        """Populates the provides that naturally apply to a user, such as being the admin of their
```
Comment (kurtismullins) [inline auth/permissions.py]: Another multi-line miss

File: auth/signedgrant.py [hunk 4]
```diff
@@ -24,9 +24,8 @@ def generate_signed_token(grants, user_context):
 
 
 def validate_signed_grant(auth_header):
-    """ Validates a signed grant as found inside an auth header and returns whether it points to
-      a valid grant.
-  """
+    """Validates a signed grant as found inside an auth header and returns whether it points to a
```
Comment (kurtismullins) [inline auth/signedgrant.py]: Multi-line miss

File: avatars/avatars.py [hunk 5]
```diff
@@ -24,17 +24,16 @@ def __getattr__(self, name):
...
     def __init__(self, preferred_url_scheme, colors, http_client):
         self.preferred_url_scheme = preferred_url_scheme
         self.colors = colors
         self.http_client = http_client
 
     def get_mail_html(self, name, email_or_id, size=16, kind="user"):
-        """ Returns the full HTML and CSS for viewing the avatar of the given name and email address,
-        with an optional size.
-    """
+        """Returns the full HTML and CSS for viewing the avatar of the given name and email address,
```
Comment (kurtismullins) [inline avatars/avatars.py]: Multi-line miss

**Comment by alecmerdler** [comment]
@kurtis thanks for pointing those out; let me mess around with the `docformatter` options and see what we can do.

**Comment by alecmerdler** [comment]
@kurtismullins Fixed the issues by using `--make-summary-multi-line` flag. Also added `docformatter` to our `lint` CI task. 

@josephschorr The `black` tool won't touch docstrings; it only [messed them initially because of indentation](https://github.com/psf/black/issues/144#issuecomment-469558941).

**Comment by kurtismullins** [comment]
@alecmerdler Heads up -- It looks like the tests failed on this PR with a segmentation fault.

**Comment by kurtismullins** [comment]
@alecmerdler If you want to temporarily remove the command from `Makefile` until we fix the segmentation fault, and rebase this against master to resolve conflicts, it looks good to me. Great job!

**Comment by alecmerdler** [comment]
@kurtismullins Removed the commit with `Makefile` changes and rebased.

**Comment by kurtismullins** [comment]
> @kurtismullins Removed the commit with `Makefile` changes and rebased.

Awesome, thank you! As soon as the tests go green, I'll click on the approve button. I don't see any reason they should fail, though.

**Pull request merged**